### DISCUSSION TITLE
Linux Download link is broken

### DISCUSSION THREAD

**Discussion by dcipher** [webui-dev/webui#151] [General]
Hi

I tried to download the linux version from WebUI.me and I get a 404 error.  If I download from GIT the linux archive will not open because an error occurs.

Simon

**Comment by dcipher** [comment]
I should clarify that the Linux download does open on Windows using BandiZip but the Archive Manager in OpenSuse 15.3 experiences and error when opening the zip file.

**Comment by AlbertShown** [comment]
Based on this [closed issue](https://github.com/webui-dev/webui/issues/120), I guess you should change the extension from `.zip` to `tar.gz`. This typo will be fixed in the next coming version 2.0.4. Thank you for pointing this out 👍

**Comment by ttytm** [comment]
@dcipher do the zip archives that are uploaded in the actions work for you https://github.com/webui-dev/webui/actions/runs/5630906340?pr=144?

**Reply by dcipher** [reply]
Yes they do.

**Reply by ttytm** [reply]
Okay great, thanks for the info! We'll use them for the releases soon.
